rmgpy/cantherm/qchem.py

```diff
@@ -138,9 +149,6 @@ def loadGeometry(self):
                 line = f.readline()
             # Read the next line in the file
             line = f.readline()
-            if found ==1: break
-        line = f.readline()
-        #print coord
         f.close()
         coord = numpy.array(coord, numpy.float64)
         mass = numpy.array(coord, numpy.float64)
```

Comment:
This line can be removed. Very curious as to why it even exists.

Reply:
I agree this is strange. But needs to be double-checked since the function returns `mass` (and we check that it's not empty)

Reply:
`mass` gets set in the lines after this one, so the `mass` in this line can be removed without issue.